ibm/service/power/resource_ibm_pi_spp_placement_group.go

```diff
@@ -0,0 +1,152 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	st "github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/helpers"
+	models "github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+)
+
+func ResourceIBMPISPPPlacementGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMPISPPPlacementGroupCreate,
+		ReadContext:   resourceIBMPISPPPlacementGroupRead,
+		UpdateContext: resourceIBMPISPPPlacementGroupUpdate,
+		DeleteContext: resourceIBMPISPPPlacementGroupDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
+			Update: schema.DefaultTimeout(60 * time.Minute),
+			Delete: schema.DefaultTimeout(60 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			Arg_SPPPlacementGroupName: {
```

Comment:
since we don't have update can you ForceNew : true for Required /optional

Reply:
Done